source/hardware/controllers/numark_party_mix.rst

```diff
@@ -0,0 +1,118 @@
+Numark Party Mix
+================
+
+`Manufacturer's product page <https://www.numark.com/product/party-mix>`_ · `Manufacturer's user manual <https://www.numark.com/images/product_downloads/Party_Mix_-_User_Guide_-_v1.1.pdf>`_ · `Forum thread <https://mixxx.discourse.group/t/numark-party-mix-midi-mapping/16712>`_
+
+.. versionadded:: 2.3.3
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/numark_party_mix.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Numark Party Mix
+   :figclass: pretty-figures
+
+See the tables below for controls descriptions.
+
+Center
+------
+
+..
+
+.. csv-table::
+   :header: "#", "Name", "Function"
+   :widths: 5 25 70
+
+   "1", ":hwlabel:`BROWSE` (rotate)", "Browse library"
+   "1", ":hwlabel:`BROWSE` (press+rotate)", "switch between Library and Sidebar"
```

Comment:
```suggestion
   "1", ":hwlabel:`BROWSE` (press+rotate)", "switch between Sidebar, Library and Searchbar"
```

Reply:
@ronso0 this is not the case on 2.3.2

Reply:
but maybe it will be on 2.3.3?

Reply:
oh, yes, didn't use 2.3 lately : ) 
Though, that is partially true: the **empty** searchbar can not be focused.
As soon as some text is typed the **Clear** button is displayed which can be focused (and triggered with Browse push.

You're right, let's leave it as it is, users will figure that out themselves